atlassian/bitbucket/cloud/repositories/__init__.py

```diff
@@ -62,6 +62,23 @@ def each(self, after=None, role=None, q=None, sort=None):
         for repository in self._get_paged(None, params):
             yield self._get_object(repository)
 
+    def get(self, workspace, repo_slug):
+        """
+        Returns the requested repository.
+
+        Since this method accesses the repository endpoint
+        directly it is usable if you do not have permission
+        to access the workspace endpoint.
+
+        :param workspace: string: The workspace of the repository
+        :param repo_slug: string: The requested repository.
+
+        :return: The requested Repository object
+
+        API docs: https://developer.atlassian.com/cloud/bitbucket/rest/api-group-repositories/#api-repositories-workspace-repo-slug-get
+        """
+        return self._get_object(super(Repositories, self).get("{}/{}".format(workspace, workspace)))
```

Comment:
After the changes it is now broken:

```suggestion
        return self._get_object(super(Repositories, self).get("{}/{}".format(workspace, repo_slug)))
```